apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -148,6 +148,19 @@ statements(pgsql) ->
       <<"UPDATE groups SET last_updated_by= $1, updated_at= $2, name= $3"
         "WHERE id= $4">>},
      {delete_group_by_id, <<"DELETE FROM groups WHERE id= $1">>},
+     {list_policies_for_org, <<"SELECT name, policy_group FROM policies WHERE org_id= $1">>},
+     {find_policy_by_orgid_name_group,
+      <<"SELECT id, authz_id, org_id, name, last_updated_by, serialized_object"
+        " FROM policies"
+        " WHERE (name = $1 AND policy_group = $2 AND org_id = $3) LIMIT 1">>},
```

Comment:
I'm not a fan of limit 1 when we only expect 1 result - only b/c it would mask an actual problem if we ever got more than one.